### ISSUE TITLE
setting colors on TRADFRI WS opal bulbs

### ISSUE THREAD

**Issue by thkl** [AlCalzone/node-tradfri-client#13]
Hi, setting the color temperature of WP opal bulbs is currently not possible. There are 3 color's available in the IKEA App (#efd275 ; #f1e0b5 and #f5faf6) . I was not able to set colors via setColor (throws an error cause the bulb is not an rgb device) and via setColorTemperature (just nothing happens). 

i just played around with that bulb and it turns out setting color to 'warm','normal' and 'cold' works well.

**Comment by AlCalzone** [comment]
Hey,
`setColor()` is not supposed to work, although the 3 predefined colors would make sense. I can add that exception.
`setColorTemperature()` should work indeed (the predefined colors should equal 0, 67 and 100%). I'm going to check what's going wrong here.

**Comment by thkl** [comment]
setColorTemperature() with 0 (cold white) is working .. but the others (67 and 100) do not change anything

**Comment by AlCalzone** [comment]
> the following works:
```
cold white -> operateLight({colorTemperature: 0},0.5)
normal white -> operateLight({colorTemperature: 367},0.5)
warm white -> operateLight({colorTemperature: 450},0.5)
```

That shouldn't be. The function should accept 0-100% values like the properties on the objects, not the internal color temperature in Mireds. The other way also limits the settable values to 3 distinct ones instead of the whole spectrum.

**Comment by thkl** [comment]
> I'm working on it.

ok, thank you

**Comment by AlCalzone** [comment]
fixed in v0.5.1
